### ISSUE TITLE
Namespaces can not be terminated after `kfctl delete`

### ISSUE THREAD

**Issue by Jeffwan** [kubeflow/kubeflow#4778]
/kind bug

**What steps did you take and what happened:**
I try to include cert-manager namespace and crds deletion in kfctl. The problem is after `kfctl delete ` , I notice resource objects have been deleted in all namespaces but these three namespaces are alway in `Terminating` status. 

```
➜  ~ kubectl get ns
NAME              STATUS        AGE
anonymous         Active        31m
cert-manager      Terminating   11h
default           Active        12h
istio-system      Terminating   11h
kube-node-lease   Active        12h
kube-public       Active        12h
kube-system       Active        12h
kubeflow          Terminating   11h
```

Checking controller-manager logs, I notice there's some cluster level webhook still there. 

```
E0218 19:20:26.662720       1 memcache.go:199] couldn't get resource list for webhook.cert-manager.io/v1beta1: the server is currently unable to handle the request
```

In order to unblock namespace deletion. We have to delete api service `v1beta1.webhook.cert-manager.io`.

` kubectl delete apiservices v1beta1.webhook.cert-manager.io` will work.
If not, please try `kubectl delete ns cert-manager istio-system kubeflow --force --grace-period 0`.

In conclusion, if user want to delete cert-manager, please check [manifest](https://github.com/kubeflow/manifests/tree/master/cert-manager) and delete all of them. Otherwise, there's no backend for the webhook and cluster won't work as expect.

**What did you expect to happen:**
Namespace should be deleted successfully. 

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Kubeflow version: (version number can be found at the bottom left corner of the Kubeflow dashboard): v1.0
- kfctl version: (use `kfctl version`): master
- Kubernetes platform: (e.g. `minikube`): AWS EKS
- Kubernetes version: (use `kubectl version`): 1.14
- OS (e.g. from `/etc/os-release`):

**Comment by Jeffwan** [comment]
This is not in current kfctl version and this issue has been resolve. Just want to surface this incase some one will meet similar problem

**Comment by Jeffwan** [comment]
Similarily, for `knative-serving`

```shell
v1beta1.coordination.k8s.io                 Local                               True                      25h
v1beta1.custom.metrics.k8s.io               knative-serving/autoscaler          False (ServiceNotFound)   4h38m
v1beta1.events.k8s.io                       Local                               True                      25h

```
